WorkloadManagementSystem/DB/PilotsLoggingDB.py

```diff
@@ -77,7 +77,7 @@ def __initializeDB( self ):
 
     if not 'PilotsLogging' in tablesInDB:
       try:
-        PilotsLogging.__table__.create( self.engine )
+        PilotsLogging.__table__.create( self.engine ) #pylint: disable=no-member
```

Comment:
Please remove this one: I fixed, together with some more fixes, in #3497

Reply:
Dropped commit